README.md

```diff
@@ -34,6 +34,8 @@ Once imported, dhis2 selection filters can be called in as
     <ngx-dhis2-selection-filters
       (filterUpdate)="onFilterUpdateAction($event)"
       [dataSelections]="dataSelections"
+      [disablePeriodTypeSelection]="false"
+      [selectedPeriodType]="'BiMonthly'"
```

Comment:
selectedPeriodType should part of periodFilterConfig contained in selectionFilterConfig